uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/performance/MemoryMonitor.java

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.performance;
+
+/**
+ * Collect memory information regarding the current process. This will basically return the total memory being used in KiloBytes
+ */
+public class MemoryMonitor implements PerformanceProfileMonitor<Long> {
+    @Override
+    public Long getStats(ProcessInfo processInfo) {
+        return (long) (processInfo.getMemoryUsage() * processInfo.getUsedSystemMemory()) / 100;
```

Comment:
What does `processInfo.getMemoryUsage()` return here? and what do we get when we multiply that with `processInfo.getUsedSystemMemory()`...I guess my question is...what is the difference between these two?

Reply:
I should add some comments. 
`usedSystemMemory` is the memory used by all the processes in the device.
`memoryUsage` is the percentage of memory being used by the specific device.